skbeam/core/roi.py

```diff
@@ -96,21 +96,23 @@ def rectangles(coords, shape):
 
 def rings(edges, center, shape):
     """
-    Draw annual (ring-shaped) regions of interest.
+    Draw annual (ring-shaped) or bar shaped regions of interest.
 
-    Each ring will be labeled with an integer. Regions outside any ring will
+    Each roi will be labeled with an integer. Regions outside any roi will
     be filled with zeros.
 
     Parameters
     ----------
     edges: list
-        giving the inner and outer radius of each ring
+        giving the inner and outer radius of each roi (ring or bar)
         e.g., [(1, 2), (11, 12), (21, 22)]
-
-    center : tuple
-        point in image where r=0; may be a float giving subpixel precision.
-        Order is (rr, cc).
-
+    center : tuple or array
+        1. tuple
+           point in image where r=0; may be a float giving subpixel precision.
+           Order is (rr, cc).
+        2. array
+           image pixels co-ordinates
+           shape image shape
```

Comment:
I do not understand the distinction you are making between `tuple` and `array`. Is the expected value different if you pass in a `tuple` versus an `array`?